docs/devel/cherry-picks.md

```diff
@@ -39,7 +39,31 @@ Kubernetes projects.
 
 ## Propose a Cherry Pick
 
-Any contributor can propose a cherry pick of any pull request, like so:
+When the release-1.2 branch was created, we shifted to a new cherrypick model
+where the branch 'OWNERS' cherry pick batches of patches into the branch
+to control the order and also vet what is or is not cherrypicked to a branch.
+
+Contributors that want to include a cherrypick for a branch should label
+their PR with the `cherrypick-candidate` label.  These cherrypick-candidate's
```

Comment:
label AND a milestone (or the bot will unlabel).  In the case of a released milestone, that milestone will be listed in red (overdue) on github (sort of obnoxious).